EnvironmentSetup/AWS/Source/deployments/active-web-elements-server-deployment.yaml

```diff
@@ -74,7 +74,8 @@ spec:
             path: /.applicationserver/info
             port: 8080
           initialDelaySeconds: 15
-          periodSeconds: 20
+          periodSeconds: 200
```

Comment:
You shouldn't increase the **periodSeconds**, as this sets how often the probe will be tried. By setting it to 200 it will only check every seconds which is too slow. The original 20 was fine. By setting the failure threshold to 10 it will try 10 times every 20 seconds which gives the pod a full 200 seconds to go live. That is probably sufficient.

Reply:
Fixed